src/components/input-form/InputForm.component.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import InputElem from '../input-elem/InputElem.component';
+import InputSection from '../input-section/InputSection.component';
+import AddSection from '../AddSection/AddSection.component.js';
+
+import './input-form.styles.css';
+
+class InputForm extends React.Component{
+  constructor(props){
+    super(props)
+
+    this.state = {
```

Comment:
it makes so much sense to do this in react with the state handling capabilities, very smart